lib/iris/tests/test_image_json.py

```diff
@@ -80,7 +79,12 @@ def test_resolve(self):
         exceptions = deque()
         uri_queue = Queue()
         for uri in uris:
-            uri_queue.put(uri)
+            if uri.startswith('https://scitools.github.io'):
+                uri_queue.put(uri)
+            else:
+                msg = '{} is not a valid resource.'.format(uri)
+                exceptions.append(ValueError(msg))
+                continue
```

Comment:
what is the `continue` here for please, I am not convinced it is doing anything active